static-site/src/sections/sars-cov-2-page.jsx

```diff
@@ -143,9 +159,27 @@ class Index extends React.Component {
     }
   }
 
+  banner() {
+    let bannerTitle, bannerContents;
+    if (this.state.nonExistentDatasetName) {
+      bannerTitle = `The dataset "nextstrain.org/ncov/${this.state.nonExistentDatasetName}" doesn't exist.`;
```

Comment:
[notes from zoom call yesterday] banner message:

The dataset "nextstrain.org${this.props.uri}/${this.state.nonExistentDatasetName}" doesn't seem to exist. Please <a href="https://discussion.nextstrain.org/">post on the discussion board</a> if you believe there has been an error here.

Below is our main SARS-CoV-2 page which lists all current datasets & narratives which we are aware of.